DDSim/DDSim/DD4hepSimulation.py

```diff
@@ -618,6 +623,41 @@ def __addParametersToRunHeader( self ):
 
     return runHeader
 
+
+  def __addParametersToEvent(self, parameters):
```

Comment:
Please move this function to `Meta`, and maybe call it `getEventParameters` or `parseEventParameters`